deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/OafResource.java

```diff
@@ -94,8 +95,18 @@ public class OafResource implements Resource {
     private OafDatasetConfiguration oafConfiguration;
 
     private HtmlViewConfiguration htmlViewConfiguration;
-
-    public OafResource( ResourceMetadata<Resource> metadata, Workspace workspace, DeegreeOAF config ) {
+    
+    private Collection additionalCollection;
+    
+	public Collection getAdditionalCollection() {
```

Comment:
I have no idea why the formatting is shifted in this file and the other files. Do you have set any style configuration? In my eclipse and also in the desktop git hub the formatting looks proper. 
![Screenshot 2023-03-15 at 12 11 22](https://user-images.githubusercontent.com/28185404/225292393-7f4aa12b-3f20-40cc-9ac8-8b6f37c0bf64.png)

Reply:
Usually that is a tabs vs spaces problem? And then shows up depending on the assumed tab length

Reply:
Yes you are absolutely right, tabs was tabs problem. I have now corrected it. 
Also, I have reverted the change which displayed datasetId in the dataset title page as I thought and was not the correct place to do the change. Ideally `Links` should say `Landing Page as HTML` and if possible the dataset ID should replace `deegree OGC API - Features` title in the `Example Service` / or the landing Page. I will check if I can find and create a PR for the same.